views/add-ons.php

```diff
@@ -47,14 +47,21 @@
 						<?php TemplateTags\list_topics( $addon['topics'], 3, $addon['url'] ); ?>
 					<?php endif; ?>
 
-					<p class="wp101-addon-button">
-						<?php if ( $has_addon ) : ?>
+					<?php if ( $has_addon && $addon['meets_requirements'] ) : ?>
```

Comment:
`$addon['meets_requirements']` *should* always be true or false, but make this future proof by explicitly checking for a true or non-empty value.